sirepo/package_data/template/radia/examples/wiggler.json

```diff
@@ -5,8 +5,9 @@
             "paths": []
         },
         "geometry": {
+            "isSolvable": "0",
```

Comment:
Not sure if this example is out in the wild but if so - maybe modify sim_data.radia.fixup_old_data to check if the data to fixup is the wiggler example and if isSolvable isn't found set it to 0. Currently it will be set to the default which is 1.

Reply:
It almost certainly is not in the wild, but it's probably best practice to assume it is

Reply:
1a3ae83